app/code/Vndr/Mod/view/frontend/layout/default.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="header-wrapper">
+            <block class="Magento\Framework\View\Element\Template" name="top.search" as="topSearch" template="Vndr_Mod::form.mini.phtml">
```

Comment:
@yuratouch You should use instruction`<referenceBlock>` for existed block. And please, remove redundant `<referenceContainer>`. Also good approach for template name is to use dash instead of dot

Reply:
Please, explain the difference between usage <referenceBlock> and <referenceContainer> because practically I didn't see any differences.